docs/using_tags.md

```diff
@@ -149,27 +160,64 @@ E.g. on Move Location signal the following tags will be purged:
-            'path-' . $signal->locationId,
+            'p' . $signal->locationId,
             // old parent
-            'location-' . $signal->oldParentLocationId,
+            'l' . $signal->oldParentLocationId,
             // old siblings
-            'parent-' . $signal->oldParentLocationId,
+            'pl' . $signal->oldParentLocationId,
             // new parent
-            'location-' . $signal->newParentLocationId,
+            'l' . $signal->newParentLocationId,
             // new siblings
-            'parent-' . $signal->newParentLocationId,
+            'pl' . $signal->newParentLocationId,
         ];
     }
 ```
 
-All slots can be found in `src/SignalSlot`.
-
+All slots can be found in `src/SignalSlot`
 
 ## Troubleshooting
 
-One common issue to encounter is that the tagging headers exceed limits in Varnish, stopping either caching or invalidation from happening:
-- [http_resp_hdr_len](https://varnish-cache.org/docs/6.0/reference/varnishd.html#http-resp-hdr-len) (e.g. 32k)
-- [http_max_hdr](https://varnish-cache.org/docs/6.0/reference/varnishd.html#http-max-hdr) (e.g. 128)
-- [http_resp_size](https://varnish-cache.org/docs/6.0/reference/varnishd.html#http-resp-size) (e.g. 64k)
+### Understanding tag response headers
+
+Given a header like `Xkey: 0c22 0ct2 (...)`, the following can be understood:
+- `Xkey:` Configured tag header, by this bundle config. Understood by Varnish _(with xkey module)_, and by our enhanced version of Symfony HttpCache Proxy "AppCache".
+- `0..`: Current repository index, set for you by this bundle. Done in order to support multi repository setups, it's the array key of the current repository is used _(and not the full identifier)_.
+- `..c22`: Content id 22
+- `..ct2`: Content Type id 2
+
+### Header limits
+
+Even if the tags are kept as short as possible, you might still encounter issue with tag header exceeding
+limits in Varnis/Apache/Nginx/Fastly, stopping either caching or invalidation from working as expected.
+
+For handling these cases it's simplest to increase the limits on the service side if possible, and if not look at the
+code involved to see if amount of tags can be reduced.
+
+#### Configuring Services
+
+*Varnish* config:
+- [http_resp_hdr_len](https://varnish-cache.org/docs/6.0/reference/varnishd.html#http-resp-hdr-len) (default 8k, change to i.e. 32k)
+- [http_max_hdr](https://varnish-cache.org/docs/6.0/reference/varnishd.html#http-max-hdr) (default 64, change to i.e. 128)
+- [http_resp_size](https://varnish-cache.org/docs/6.0/reference/varnishd.html#http-resp-size) (default 23k, change to i.e. 96k)
+- [workspace_backend](https://varnish-cache.org/docs/6.0/reference/varnishd.html#workspace-backend) (default 64k, change to i.e. 128k)
+
+*Fastly* has a `Surrogate-Key` header limit of *16kb*, this can *not* be changed.
+
+*Apache* has a limit of 8kb, and seems to need to be re-compiled to change this TODO: Find link about this
```

Comment:
As a ref: https://github.com/apache/httpd/blob/7e2d26eac309b2d79e467ef586526c10e0f226f8/include/httpd.h#L300
https://github.com/apache/httpd/blob/5f32ea94af5f1e7ea68d6fca58f0ac2478cc18c5/server/util_script.c#L495

Reply:
Thanks, added in 679a74d 👍